mkosi

```diff
@@ -350,7 +350,7 @@ def umount(where):
     # Ignore failures and error messages
     subprocess.run(["umount", "-n", where], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
 
-def prepare_tree(args, workspace):
+def prepare_tree(args, workspace, run_build_script):
```

Comment:
Shouldn’t the `prepare_tree` call in line 1587 also be updated to propagate the `run_build_script` parameter?